commandapi-kotlin/src/test/kotlin/Examples.kt

```diff
@@ -0,0 +1,87 @@
+import dev.jorel.commandapi.*
+import dev.jorel.commandapi.arguments.ArgumentSuggestions
+import dev.jorel.commandapi.arguments.LiteralArgument.of
+import dev.jorel.commandapi.arguments.StringArgument
+import org.bukkit.Bukkit
+import org.bukkit.command.CommandSender
+import org.bukkit.entity.Player
+import org.bukkit.inventory.ItemStack
+
+fun sendMessageToCommand() {
+/* ANCHOR: dslSendMessageToCommand */
+command("sendmessageto") {
+    playerArgument("player") { // Defines a new PlayerArgument("player")
+        greedyArgument("msg") { // Defines a new GreedyStringArgument("msg)
+            anyExecutor { _, args -> // Command can be executed by anyone and anything (such as entities, the console, etc.)
+                val player: Player = args[0] as Player
+                val message: String = args[1] as String
+                player.sendMessage(message)
+            }
+        }
+    }
+}
+/* ANCHOR_END: dslSendMessageToCommand */
+
+/* ANCHOR: dslSendMessageToCommandRequirement */
+command("sendMessageTo") {
+    playerArgument("player") {
+        greedyArgument("msg") {
+            playerExecutor { _, args ->
+                val player: Player = args[0] as Player
+                val message: String = args[1] as String
+                player.sendMessage(message)
+            }
+        }
+    }
+    requirement(of("broadcast"), { sender: CommandSender -> (sender is Player) && sender.isOp }) { // Define a new LiteralArgument("broadcast") that requires the CommandSender to be a player who is a server operator
```

Comment:
The check for whether the sender is a player is redundant, because this command only uses `playerExecutor`. The CommandAPI documentation [already has a similar case](https://commandapi.jorel.dev/8.5.1/tooltips.html#example---teleportation-command-with-suggestion-descriptions) where we already know the cast will work as long as we use `.executesPlayer`.